### PR TITLE
[Master] Update supported cipher suites in product documentations

### PR DESCRIPTION
**Pull request by divyaamunugama**
## Purpose
$subject

Related Issue: https://github.com/wso2/product-is/issues/15325

### CODE REVIEW

File: en/docs/deploy/security/configure-transport-level-security.md [hunk 1]
```diff
@@ -25,130 +24,90 @@ Follow the instructions given below to enable SSL protocols in the WSO2 Identity
...
+    <Ciphers>Ciphername1, Ciphername2</Ciphers>
```
Comment (divyaamunugama) [inline en/docs/deploy/security/configure-transport-level-security.md:88]: This is the update related to the issue

File: en/docs/deploy/security/configure-transport-level-security.md [hunk 2]
```diff
@@ -25,130 +24,90 @@ Follow the instructions given below to enable SSL protocols in the WSO2 Identity
...
+    ciphers="<cipher-name>,<cipher-name>"
```
Comment (divyaamunugama) [inline en/docs/deploy/security/configure-transport-level-security.md:44]: This is the update related to the issue

File: en/docs/deploy/security/configure-transport-level-security.md [hunk 3]
```diff
@@ -25,130 +24,90 @@ Follow the instructions given below to enable SSL protocols in the WSO2 Identity
...
-1.  Navigate to the ` deployment.toml ` file in the ` <IS_HOME>/repository/conf   ` directory.
+2. Take a backup of the `deployment.toml` file and stop the WSO2 IS server.
 
-2.  Take a backup of the ` deployment.toml ` file and stop the WSO2 IS server.
+3. Add the following configuration to the `deployment.toml` file by adding the list of ciphers that you want your server to support as follows:
 
-3.  Add the following configuration to the ` deployment.toml ` file by adding the list of ciphers 
-that you want your server to support as follows: ciphers="<cipher-name>,<cipher-name>".
+    !!! note
+        For a list of cipher suites that are secure and functional in Tomcat for the TLSv1.2 protocols, see the list of ciphers provided in the [secure configuration generator](https://ssl-config.mozilla.org/#server=tomcat&version=9.0.30&config=intermediate&guideline=5.6), which the Mozilla Foundation provides.
```
Comment (divyaamunugama) [inline en/docs/deploy/security/configure-transport-level-security.md:40]:
```suggestion
        For a list of cipher suites that are secure and functional in Tomcat for the TLSv1.2 protocols, see the list of ciphers provided in the [secure configuration generator](https://ssl-config.mozilla.org/#server=tomcat&version=9.0.58&config=intermediate&guideline=5.6), which the Mozilla Foundation provides.
```

File: en/docs/deploy/security/configure-transport-level-security.md [hunk 4]
```diff
@@ -25,130 +24,90 @@ Follow the instructions given below to enable SSL protocols in the WSO2 Identity
...
-Follow the instructions given below to enable SSL protocols and ciphers in `         ThriftAuthenticationService.        `
+Follow the instructions given below to enable SSL protocols and ciphers in `ThriftAuthenticationService`
+
+1. Add the following configurations by adding the supported cipher suites as a comma-separated list in the `<CARBON_SERVER>/repository/conf/identity/thrift-authentication.xml` file as sub-elements of the root `<Server>` element.
 
-1.  Add the following configurations in the
-    `           <CARBON_SERVER>/repository/conf/identity/thrift-authentication.xml          `
-    file as sub-elements of the root `           <Server>          ` element.
+    !!! note
+        For a list of cipher suites that are secure and functional in Tomcat for the TLSv1.2 protocols, see the list of ciphers provided in the [secure configuration generator](https://ssl-config.mozilla.org/#server=tomcat&version=9.0.30&config=intermediate&guideline=5.6), which the Mozilla Foundation provides.
```
Comment (divyaamunugama) [inline en/docs/deploy/security/configure-transport-level-security.md:84]:
```suggestion
        For a list of cipher suites that are secure and functional in Tomcat for the TLSv1.2 protocols, see the list of ciphers provided in the [secure configuration generator](https://ssl-config.mozilla.org/#server=tomcat&version=9.0.58&config=intermediate&guideline=5.6), which the Mozilla Foundation provides.
```

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**